### PR TITLE
[STAND-BY] Feature/new api server

### PR DESCRIPTION
**Pull request by valpinkman**
/!\ Merge only when Manager API is deployed

### CODE REVIEW

File: src/components/ManagerPage/AppsList.js [hunk 1]
```diff
@@ -43,27 +44,17 @@ const ICONS_FALLBACK = {
...
   targetId: string | number,
   t: T,
+  version: string,
 }
 
 type State = {
   status: Status,
   error: string | null,
-  appsList: LedgerApp[],
+  appsList: LedgerScriptParams[] | Array<*>,
```
Comment (meriadec) [inline src/components/ManagerPage/AppsList.js]: is it intended to have uncertain type here?

File: src/components/ManagerPage/FlashMcu.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  state = {
+    flashing: false,
+  }
+
+  flashMCU = async () => {
+    const { device, deviceInfo } = this.props
+    const { flashing } = this.state
+
+    if (!flashing) {
+      this.setState(state => ({ ...state, flashing: true }))
```
Comment (meriadec) [inline src/components/ManagerPage/FlashMcu.js]: no need to `...state`

File: src/components/ManagerPage/FlashMcu.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      await installMcu
+        .send({
+          devicePath: device.path,
+          targetId: deviceInfo.targetId,
+          version: deviceInfo.version,
+        })
+        .toPromise()
+      this.setState(state => ({ ...state, flashing: false }))
```
Comment (meriadec) [inline src/components/ManagerPage/FlashMcu.js]: same

File: src/components/ManagerPage/FlashMcu.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <h1>Flashing MCU</h1>
```
Comment (meriadec) [inline src/components/ManagerPage/FlashMcu.js]: need i18n

File: src/components/ManagerPage/FlashMcu.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        <button onClick={this.flashMCU}>flash</button>
```
Comment (meriadec) [inline src/components/ManagerPage/FlashMcu.js]: same (I guess it's not finished so maybe not important)

File: src/components/ManagerPage/AppsList.js (hunk 1 above)
Comment (valpinkman) [reply src/components/ManagerPage/AppsList.js]: it was bitchin if my array was empty...

File: src/components/ManagerPage/FlashMcu.js (hunk 5 above)
Comment (valpinkman) [reply src/components/ManagerPage/FlashMcu.js]: screen reworks today

File: src/components/ManagerPage/FlashMcu.js (hunk 4 above)
Comment (valpinkman) [reply src/components/ManagerPage/FlashMcu.js]: screen reworks today

File: src/components/ManagerPage/FlashMcu.js (hunk 2 above)
Comment (valpinkman) [reply src/components/ManagerPage/FlashMcu.js]: done

File: src/components/ManagerPage/FlashMcu.js (hunk 3 above)
Comment (valpinkman) [reply src/components/ManagerPage/FlashMcu.js]: done

File: src/components/ManagerPage/AppsList.js (hunk 1 above)
Comment (meriadec) [reply src/components/ManagerPage/AppsList.js]: very strange :/ do you still reproduce?

**Pull request merged**